pkg/kubectl/describe/versioned/describe.go

```diff
@@ -1203,6 +1205,15 @@ func printFlockerVolumeSource(flocker *corev1.FlockerVolumeSource, w PrefixWrite
 		flocker.DatasetName, flocker.DatasetUUID)
 }
 
+func printCSIVolumeSource(csi *corev1.CSIVolumeSource, w PrefixWriter) {
```

Comment:
do we make kubectl describe changes for alpha fields?  could we have client incompatibility issues if we decide to change anything in a future release?

Reply:
See response in newer PR here - https://github.com/kubernetes/kubernetes/pull/74086#issuecomment-464188838